### PR TITLE
CeruleanStudios,LLC.Trillian version 6.4.0.5

### PR DESCRIPTION
**Pull request by cezarandreescu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15886)

### CODE REVIEW

**Comment by KaranKad** [comment]
Change `PackageIdentifier` and folder name to `CeruleanStudios.Trillian`

File: manifests/c/CeruleanStudios,LLC/Trillian/6.4.0.5/CeruleanStudios,LLC.Trillian.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: CeruleanStudios,LLC.Trillian
```
Comment (jedieaston) [inline manifests/c/CeruleanStudios,LLC/Trillian/6.4.0.5/CeruleanStudios,LLC.Trillian.installer.yaml:3]: Duplicate of Trillian.Trillian.

**Review by jedieaston — CHANGES_REQUESTED** [review]
This is a duplicate of Trillian.Trillian, and as such should not be merged.

Consider using `winget search` next time to see if it's worth the trouble of building an entirely new manifest. In fact, it needs an update, so if you'd be willing, can you open a PR for that instead? Thanks!

File: manifests/c/CeruleanStudios,LLC/Trillian/6.4.0.5/CeruleanStudios,LLC.Trillian.installer.yaml (hunk 1 above)
Comment (cezarandreescu) [reply manifests/c/CeruleanStudios,LLC/Trillian/6.4.0.5/CeruleanStudios,LLC.Trillian.installer.yaml:3]: PR for updating existing manifest has been created: https://github.com/microsoft/winget-pkgs/pull/15988
Sorry for the confusion.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please close the PR as the other PR have been merged :)

**Comment by OfficialEsco** [comment]
Uhhh @denelon there is a bug in the validation process 👀

**Comment by denelon** [comment]
@OfficialEsco I'll let the team know right away.

**Comment by denelon** [comment]
@OfficialEsco we're making some changes to reduce the need for "Validation-Completed" and "AzurePipelines-Passed" to both be present. It's optimization work for some parallel activities happening in the background.

**Pull request merged**